dspace/config/submission-forms.xml

```diff
@@ -188,6 +188,23 @@
         </form>
 
         <form name="traditionalpagetwo">
+            <row>
+                <relation-field>
+                    <relationship-type>isJournalIssueOfPublication</relationship-type>
+                    <search-configuration>publicationIssue</search-configuration>
+                    <label>Journal Issue</label>
+                    <hint>Select the journal issue related to this publication.</hint>
+                </relation-field>
+            </row>
+            <row>
+                <relation-field>
+                    <relationship-type>isOrgUnitOfPublication</relationship-type>
+                    <search-configuration>organization</search-configuration>
+                    <repeatable>true</repeatable>
+                    <label>Organization</label>
+                    <hint>Select the Organizational Unit related to this publication.</hint>
+                </relation-field>
+            </row>
```

Comment:
While I'm not against having these enabled for testing purposes, I'm hesitant to leave them in the main configuration by default. What would happen if these relationship types are *not* installed in the local database?  I'm assuming this configuration would be invalid & throw errors...which means this may break our demo site and Docker deployments (unless those both install the relationship types in question)

Reply:
I tend to agree with @tdonohue . Default DSpace default forms shouldn't have entities dependency.

Reply:
I've removed the new config as discussed today, trimming down the PR to fixes only